docs/modules/nifi/pages/usage_guide/external_ports.adoc

```diff
@@ -0,0 +1,63 @@
+= Adding External Files to the NiFi Servers
+
+== Exposing processors
+Some NiFi processors allow external tools to connect to them to trigger workflows or send data.
+In order for this to work from outside of the Kubernetes cluster NiFi has been deployed to it is necessary to create https://kubernetes.io/docs/concepts/services-networking/service/[Service] or https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress] objects to configure Kubernetes routes for these ports.
```

Comment:
```suggestion
In order for this to work from outside the Kubernetes cluster where NiFi has been deployed to, it is necessary to create https://kubernetes.io/docs/concepts/services-networking/service/[Service] or https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress] objects to configure Kubernetes routes for these ports.
```

Reply:
I have changed it to something else the linter suggested, please have a look..

Reply:
Yeah the where doesnt have to be there but id like the comma, otherwise its hard to read

Reply:
changed